### ISSUE TITLE
NameValueList - DS site WCAG accessibility documentation

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#5014]
Add Accessibility information to the NameValueList page on the Design System site.

- [x] Assess which [WCAG rules](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.2) are applicable to the component and if they are passing/failing.
- [x] Create a pull request on the Design System site to add this information to the component page. Follow the format here: https://github.com/grommet/hpe-design-system/pull/5004/files. The finished result should look something like this: https://design-system.hpe.design/components/menu?q=menu#wcag-compliance.
- [x] Create issues in the relevant repository (grommet, grommet-theme-hpe, etc.) for any rules where we are not meeting WCAG AA or if there is something observed that it feels like we should fix. We should create 1 issue per rule failure. Issues where we are not meeting AA should be tagged with 'bug'. All issues created through this process should have the 'accessibility' label and be added to the accessibility board.

The pull request should be up on Github within the first week of the sprint so that the second week of the sprint can be focused on review and polishing.

When your work is ready for review please get a first pass review from another member of the accessibility sprint and once that is done reach out to Jessica to flag that a final review is needed.

Resources and videos for this work can be found here: https://hpe-my.sharepoint.com/shared?id=%2Fteams%2Fhpe%2Ddesign%2FShared%20Documents%2FHPE%20Design%20System%2FAccessibility&listurl=https%3A%2F%2Fhpe%2Esharepoint%2Ecom%2Fteams%2Fhpe%2Ddesign%2FShared%20Documents

**Comment by julauxen** [comment]
First go on the audit can be found here: https://hpe.sharepoint.com/:x:/r/teams/hpe-design/Shared%20Documents/HPE%20Design%20System/Accessibility/WCAG%20Rule%20Evaluation%20Template.xlsx?d=we975e19f32644f4cbd0a446f23f89277&csf=1&web=1&e=ok8nhq

**Comment by CillianHPE** [comment]
Added some comments as a first pass.

@jcfilben @julauxen

**Comment by jcfilben** [comment]
@julauxen added comments to the excel file

**Comment by julauxen** [comment]
Thanks guys! @jcfilben I made some edits and left a few questions for you there.

**Comment by jcfilben** [comment]
@julauxen I did another pass on the excel file and hopefully answered the questions you had

**Comment by julauxen** [comment]
Thanks @jcfilben I created the PR HERE: https://github.com/grommet/hpe-design-system/pull/5054

**Comment by britt6612** [comment]
@julauxen I believe this issue can be closed as the PR was merged.

**Comment by taysea** [comment]
Closing, no grommet issues were required to be filed.
